src/azul/tdr.py

```diff
@@ -142,6 +142,10 @@ def get_access_token(self) -> str:
 class TDRClient(SAMClient):
     """
     A client for the Broad Institute's Terra Data Repository code-named Jade.
+
+    Azul currently has no need to access the TDR API, but we expect it to
+    become necessary in the future. For now, this class is only used to verify
```

Comment:
Remember to enforce [code style](https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst#1code-style) second bullet point.